.codespell/ignore_wordlist.txt

```diff
@@ -8,3 +8,4 @@ nd
 curEnt
 childRes
 specField
+runn
```

Comment:
is this needed? Isn't that a `readies` command? I don't see where you use it

Reply:
It is, still used in `runtests.sh`. Should be removed in next PRs as part of removing readies